vip-cache-manager/class-wpcom-vip-cache-manager.php

```diff
@@ -400,12 +419,14 @@ public function execute_purges() {
 		$num_ban_urls   = count( $this->ban_urls );
 		$num_purge_urls = count( $this->purge_urls );
 		if ( $num_ban_urls > self::MAX_BAN_URLS ) {
-			trigger_error( sprintf( 'vip-cache-manager: Trying to BAN too many URLs (total count %s); limiting count to %d', number_format( $num_ban_urls ), self::MAX_BAN_URLS ), E_USER_WARNING );
+			// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_trigger_error
+			trigger_error( sprintf( 'vip-cache-manager: Trying to BAN too many URLs (total count %s); limiting count to %d', number_format( $num_ban_urls ), (int) self::MAX_BAN_URLS ), E_USER_WARNING );
```

Comment:
I have added `(int)` to `self::MAX_BAN_URLS` to suppress the `WordPress.Security.EscapeOutput.OutputNotEscaped` error. phpcs does not check the format string and therefore generates false-positives.